ee/tables/systemprofiler/systemprofiler.go

```diff
@@ -102,16 +102,20 @@ func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) (
 
 	datatypeQ, ok := queryContext.Constraints["datatype"]
 	if !ok || len(datatypeQ.Constraints) == 0 {
-		return results, fmt.Errorf("The %s table requires that you specify a constraint for datatype", t.tableName)
+		return results, fmt.Errorf("the %s table requires that you specify a constraint for datatype", t.tableName)
+	}
+
+	// Check for maximum number of datatypes
+	if len(datatypeQ.Constraints) > 3 {
```

Comment:
would be nice to make the `3` a constant with a comment explaining why 3 is the max constraints

Reply:
Not sure it needs a constant, but expanding the comment makes sense...

Reply:
made it into const